tests/test_exec.py

```diff
@@ -62,11 +64,71 @@ def test_exec_not_exists():
 
 def test_exec_detach_not_exists():
     return test_exec_not_exists_helper(False)
+
+NONKM_OK_CONF = "var/lib/krun/nonkm_exec_ok_config"
+
+def test_exec_non_km_payload_helper(execpath, nonkmok):
+    # only run these tests for krun
+    runtime = os.getenv("OCI_RUNTIME")
```

Comment:
is it something that crun sets , or do we need to ?

Reply:
This is only used for the purposes of the crun tests.
If crun was being used by docker or podman then crun's path would be in /etc/docker/daemon.json for docker or /usr/share/containers/containers.conf for podman.

Reply:
so the test framwork sets it ?

Reply:
The container-runtime directory Makefile sets it:

# Run crun's tests
test:   all $(CURDIR)/$(CRUNDIR)/krun
        OCI_RUNTIME=$(CURDIR)/$(CRUNDIR)/krun make -C $(CRUNDIR) check-TESTS
        OCI_RUNTIME=$(CURDIR)/$(CRUNDIR)/crun make -C $(CRUNDIR) check-TESTS